sdk/appcenter/src/main/java/com/microsoft/appcenter/ingestion/OneCollectorIngestion.java

```diff
@@ -0,0 +1,198 @@
+package com.microsoft.appcenter.ingestion;
+
+import android.content.Context;
+import android.support.annotation.NonNull;
+import android.support.annotation.VisibleForTesting;
+
+import com.microsoft.appcenter.http.DefaultHttpClient;
+import com.microsoft.appcenter.http.HttpClient;
+import com.microsoft.appcenter.http.HttpClientNetworkStateHandler;
+import com.microsoft.appcenter.http.HttpClientRetryer;
+import com.microsoft.appcenter.http.HttpUtils;
+import com.microsoft.appcenter.http.ServiceCall;
+import com.microsoft.appcenter.http.ServiceCallback;
+import com.microsoft.appcenter.ingestion.models.Log;
+import com.microsoft.appcenter.ingestion.models.LogContainer;
+import com.microsoft.appcenter.ingestion.models.json.LogSerializer;
+import com.microsoft.appcenter.utils.AppCenterLog;
+import com.microsoft.appcenter.utils.NetworkStateHelper;
+
+import org.json.JSONException;
+
+import java.io.IOException;
+import java.net.URL;
+import java.util.HashMap;
+import java.util.LinkedHashSet;
+import java.util.Map;
+import java.util.Set;
+import java.util.UUID;
+
+import static android.util.Log.VERBOSE;
+import static com.microsoft.appcenter.AppCenter.LOG_TAG;
+import static com.microsoft.appcenter.BuildConfig.VERSION_NAME;
+import static com.microsoft.appcenter.http.DefaultHttpClient.CONTENT_TYPE_KEY;
+import static com.microsoft.appcenter.http.DefaultHttpClient.METHOD_POST;
+
+public class OneCollectorIngestion implements Ingestion {
+
+    /**
+     * Default log URL.
+     */
+    private static final String DEFAULT_LOG_URL = "https://browser.events.data.microsoft.com/OneCollector/1.0";
```

Comment:
God feedback that we should replace browser by mobile in the url, I checked that it works.

Reply:
Yes I agree should be mobile.